src/Sylius/Bundle/AdminBundle/Action/RemoveAvatarAction.php

```diff
@@ -34,9 +34,11 @@ public function __construct(
 
     public function __invoke(Request $request): Response
     {
-        $userId = $request->attributes->get('id');
+        $userId = $request->attributes->get('id', '');
```

Comment:
As idea,
```suggestion
        $userId = $request->attributes->getString('id');
```

for this and every other place with default `''` value.

https://github.com/symfony/symfony/blob/d42b5c39a5db5e65305ac30118a287501813518a/src/Symfony/Component/HttpFoundation/ParameterBag.php#L139-L147

Reply:
Unfortunately, the `getString` method is only available from Symfony 6.3. https://symfony.com/doc/current/components/http_foundation.html#accessing-request-data